CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 4.10.0
+
+* Added `created_by_name` to the response for `NotificationsAPIClient.get_notification_by_id()` and `NotificationsAPIClient.get_all_notifications()`
+    * If the notification was sent manually, this will be the name of the sender. If the notification was uploaded as a CSV or was sent through the API, this will be `None`.
```

Comment:
I don't think we need to mention CSV uploads here because the jobs are not returned by these methods